HowLong/HowLong.py

```diff
@@ -89,12 +97,12 @@ def run(self):
         process = Process(pid=self.pid,command=self.command)
         readable_command = process.command
         start_time = process.start_time
-        logging.debug("Running " + readable_command)
+        logging.debug(colored("Running " + readable_command), 'green')
```

Comment:
This needs to be `logging.debug(colored("Running " + readable_command, 'green'))`